src/vs/workbench/services/configurationResolver/common/quickInputServiceMock.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Don't just create new files in vs folder. If anything you should instead create this under the sql folder.

But as suggested elsewhere I would say it would be better to just move the test into electron-browser instead, which should fix the layering issue.